conda-env-mod

```diff
@@ -62,14 +62,13 @@ function usage()
     $(bold Optional arguments:)
            -a|--appname APP_NAME       : name of the module corresponding
                                          to the environment.
-                                         Default is '${MODULE_APP_DEF}'.
+                                         Default is '${MODULE_APP_DEF}'.	
+           -b|--base-python            : use the Python from base Anaconda module
```

Comment:
My wording would be _"use the Python from base Anaconda module (rarely needed)"_ :)